http://bugs.gentoo.org/199479

2007-09-13  H.J. Lu  <hongjiu.lu@intel.com>

	* sysdeps/i386/i586/memcpy_chk.S: New file.
	* sysdeps/i386/i586/mempcpy_chk.S: Likewise.
	* sysdeps/i386/i586/memset_chk.S: Likewise.

---
 i386/i586/memcpy_chk.S  |    1 +
 i386/i586/mempcpy_chk.S |    1 +
 i386/i586/memset_chk.S  |    1 +
 3 files changed, 3 insertions(+)

Index: eglibc-2.10/i386/i586/memcpy_chk.S
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ eglibc-2.10/i386/i586/memcpy_chk.S	2009-12-01 13:40:28.000000000 +0100
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/memcpy_chk.S>
Index: eglibc-2.10/i386/i586/mempcpy_chk.S
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ eglibc-2.10/i386/i586/mempcpy_chk.S	2009-12-01 13:40:28.000000000 +0100
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/mempcpy_chk.S>
Index: eglibc-2.10/i386/i586/memset_chk.S
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ eglibc-2.10/i386/i586/memset_chk.S	2009-12-01 13:40:28.000000000 +0100
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/memset_chk.S>
